docs/reports/v0.6/README.md

```diff
@@ -0,0 +1,3 @@
+This evaluation ran over several days and is actually assembled from several sub-evaluations. Hence, we only provide the `csv` results for now because the logs are distributed all over.
```

Comment:
Okay so instead of copying all the results we are only going to publish the CSVs.

Would be nice to have the `report` command doing all this by configuring a source/dest path. Probably a new Issue?

Reply:
Added to https://github.com/symflower/eval-dev-quality/issues/301